src/styles/main.scss

```
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import "header", "navigation", "container", "card", "reset";
```

not correct imports
1. structure of files should be different -> styles/blocks (contain "header", "navigation", "container", "card") and styles/utils (containe "reset", "variables")
2. create variables file, add there reusable colors, sizes, etc and use them in your blocks files 
for example: 
`$main-color: #060b35;`
`$small-font-size: 10px;`
`$card-horizontal-spaces: 16px;`
3. check order of properties
4. fix imports 
```suggestion
@import "./utils/reset.scss";
@import "./utils/variables.scss";
...
```